gmso/tests/files/restrained_benzene_ua.top

```diff
@@ -0,0 +1,76 @@
+; File Topology written by GMSO at 2022-10-26 16:12:59.516998
+
+[ defaults ]
+; nbfunc	comb-rule	gen-pairs		fudgeLJ	fudgeQQ
+1			2			no		0.0		0.0
+
+[ atomtypes ]
+; name		at.num	mass		charge		ptype	sigma	epsilon
+CH_sp2      6       13.01900     0.00000	A        0.36950     0.41988
+
+[ moleculetype ]
+; name	nrexcl
+Compound	3
+[ atoms ]
+; nr	type	resnr	residue		atom	cgnr	charge	mass
+1       CH_sp2      1       Compound    C       1        0.00000    13.01900
+2       CH_sp2      1       Compound    C       1        0.00000    13.01900
+3       CH_sp2      1       Compound    C       1        0.00000    13.01900
+4       CH_sp2      1       Compound    C       1        0.00000    13.01900
+5       CH_sp2      1       Compound    C       1        0.00000    13.01900
+6       CH_sp2      1       Compound    C       1        0.00000    13.01900
+
+[ bonds ]
+; ai	aj		funct	b0		kb
+2       1       1           0.14000        0.00000
+6       1       1           0.14000        0.00000
+3       2       1           0.14000        0.00000
+4       3       1           0.14000        0.00000
+5       4       1           0.14000        0.00000
+6       5       1           0.14000        0.00000
+
+[ angles ]
+; ai	aj		ak		funct	phi_0	k0
+1       2       3       1         120.00000        0.10000
+2       1       6       1         120.00000        0.10000
+1       6       5       1         120.00000        0.10000
+4       5       6       1         120.00000        0.10000
+2       3       4       1         120.00000        0.10000
+3       4       5       1         120.00000        0.10000
+
+[ angle_restraints ]
+; ai	aj		ai		ak		funct	theta_eq	k	multiplicity
+2       1       2       3       1         120.00000     1000.00000   1
```

Comment:
It looks like the first column here may be by accident; is this the case?

Reply:
I understand now from talking to @daico007 , it is comparing between 2 vectors atoms 1-2 and 3-4.